src/Microsoft.ML.Vision/ImageClassificationTrainer.cs

```diff
@@ -752,15 +752,15 @@ private void CheckTrainingParameters(Options options)
             var decodedImage4d = tf.expand_dims(decodedImageAsFloat, 0);
             var resizeShape = tf.stack(new int[] { inputDim.Item1, inputDim.Item2 });
             var resizeShapeAsInt = tf.cast(resizeShape, dtype: tf.int32);
-            var resizedImage = tf.image.resize_bilinear(decodedImage4d, resizeShapeAsInt, false, "ResizeTensor");
+            var resizedImage = tf.image.resize_bilinear(decodedImage4d, resizeShapeAsInt, false, name: "ResizeTensor");
             return (jpegData, resizedImage);
         }
 
         private static Tensor EncodeByteAsString(VBuffer<byte> buffer)
         {
             int length = buffer.Length;
-            var size = c_api.TF_StringEncodedSize((UIntPtr)length);
-            var handle = c_api.TF_AllocateTensor(TF_DataType.TF_STRING, IntPtr.Zero, 0, (UIntPtr)((ulong)size + 8));
+            var size = c_api.TF_StringEncodedSize((ulong)length);
```

Comment:
@codemzs Can you confirm the change in `EncodeByteAsString` function is correct?